.github/workflows/py-build-main.yml

```diff
@@ -0,0 +1,125 @@
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: 0 18 * * 1
+
+env:
+  CIBW_TEST_COMMAND: pytest {project}/aicspylibczi/tests
+  CIBW_TEST_EXTRAS: test
+  CIBW_SKIP: cp36-* cp310-* *-win32 pp*
+
+name: Python Build Main
+jobs:
+  python-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+        os: [ubuntu-latest, windows-2019, macOS-10.15, macOS-11.0]
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
why actions/checkout@v1 here and actions/checkout@v2 below?

Reply:
because :shrug: